SQL Assignments 2,3,4.txt

```diff
@@ -0,0 +1,458 @@
+
+--INSERT INTO products
+--	(id,title,price,description,category,company) VALUES
+
+--(05,'Apple',30000,'brand new Smartphones',01,'Apple'),
+--(06,'Notebooks',100,'All Notebooks',02,'ClassMate'),
+--(07,'Fan',200,'Cooling Fans',03,'Hawels');
+
+--INSERT INTO orders
+--	(total,date) VALUES
+--(3,16-08-2022),
+--(6,20-08-2022),
+--(15,25-08-2022);
+
+--INSERT INTO order_details 
+--	(orderId,productId,quantity,price,totalPrice,discount) VALUES
+--(01,05,5,3000,2000,50),
+--(02,06,8,100,1200,20),
+--(03,07,4,200,500,25);
+
+						---Assignment 3---
+
+
+--CREATE DATABASE sales_db;
+--USE sales_db;
+
+--CREATE TABLE salespeople(snum int, sname varchar(10), city varchar(10), comm decimal(3,2));
+
+-- INSERT INTO salespeople VALUES(1001, 'Peel', 'London', 0.12);
+-- INSERT INTO salespeople VALUES(1002, 'Serres', 'San Jose', 0.13);
+-- INSERT INTO salespeople VALUES(1004, 'Motika', 'London', 0.11);
+-- INSERT INTO salespeople VALUES(1007, 'Rifkin', 'Barcelona', 0.15);
+-- INSERT INTO salespeople VALUES(1003, 'Axelrod', 'New York', 0.10);
+
+--SELECT * FROM salespeople;
+
+--CREATE TABLE customers(cnum int, cname varchar(10), city varchar(10), rating int, snum int);
+
+-- INSERT INTO customers VALUES(2001, 'Hoffman', 'London', 100, 1001);
+-- INSERT INTO customers VALUES(2002, 'Giovanni', 'Rome', 200, 1003);
+-- INSERT INTO customers VALUES(2003, 'Liu', 'San Jose', 200, 1002);
+-- INSERT INTO customers VALUES(2004, 'Grass', 'Berlin', 300, 1002);
+-- INSERT INTO customers VALUES(2006, 'Clemens', 'London', 100, 1001);
+-- INSERT INTO customers VALUES(2008, 'Cisneros', 'San Jose', 300, 1007);
+-- INSERT INTO customers VALUES(2007, 'Pereira', 'Rome', 100, 1004);
+
+-- SELECT * FROM customers;
+
+-- CREATE TABLE orders(onum int, amt decimal(7,2), odate date, cnum int, snum int);
+
+-- INSERT INTO orders VALUES(3001,18.69,'1990-10-03', 2008, 1007);
+-- INSERT INTO orders VALUES(3003,767.19,'1990-10-03', 2001, 1001);
+-- INSERT INTO orders VALUES(3002,1900.10,'1990-10-03', 2007, 1004);
+-- INSERT INTO orders VALUES(3005,5160.45,'1990-10-03', 2003, 1002);
+-- INSERT INTO orders VALUES(3006,1098.16,'1990-10-03', 2008, 1007);
+-- INSERT INTO orders VALUES(3009,1713.23,'1990-10-04',2002, 1003);
+-- INSERT INTO orders VALUES(3007,75.75,'1990-10-04',2004, 1002);
+-- INSERT INTO orders VALUES(3008,4723.00,'1990-10-04',2006, 1001);
+-- INSERT INTO orders VALUES(3010,309.95,'1990-10-04',2004, 1002);
+-- INSERT INTO orders VALUES(3011,9891.88,'1990-10-04',2006, 1001);
+
+-- SELECT * FROM orders;
+
+-- Write a query that produces all rows from the Customers table for which the salesperson’s number
+--is 1001
+
+--SELECT * FROM customers where snum = 1001;
+
+--Write a select command that produces the rating followed by the name of each customer in San
+--Jose
+
+--select rating,cname FROM customers where city='san jose';
+
+--Write a query that will give you all orders for more than Rs. 1,000
+
+--SELECT * FROM orders WHERE amt > 1000;
```

Comment:
Its good practice to use column names instead of *

Reply:
ok sir, got it

Reply:
done with changes